tests/Koans11AboutTuples.elm

```diff
@@ -36,7 +32,10 @@ testSuite =
             \() ->
                 case ( 1, 2 ) of
                     ( first, second ) ->
-                        first == x____replace me____x && second == x____replace me____x
+                        first
+                            == x____replace me____x
```

Comment:
I don't think this `elm-format` will work for us either—the other formatting intentionally communicated the precedence. Same note applies to `Koans05AboutMathematicalOperators`.

Reply:
Will revert, both 11st and 05th koans.